src/router.ts

```diff
@@ -146,6 +164,15 @@ const introduceNewVoterToSlackChannel = async (
 
   const slackBlocks = SlackBlockUtil.getVoterStatusBlocks(operatorMessage);
 
+  // already voted?
+  const status = await DbApiUtil.getLatestVoterStatus(userInfo.userId);
+  if (status === 'ALREADY_VOTED') {
+    slackBlocks[2].elements[0].initial_option =
```

Comment:
IIUC this is assuming that the dropdown is the third block and first element, right? can you use can you use populateDropdownNewInitialValue https://github.com/tomerovadia/VoterHelpLine/blob/09f87a94b712094a9f9631e908e8d292492ba873/src/slack_block_util.ts#L459 so we're resilient to changes in the block structure down the line?

You can see an example here: https://github.com/tomerovadia/VoterHelpLine/blob/master/src/slack_interaction_handler.ts#L257

Also, you need to call replaceSlackMessageBlocks or the change does not persist https://github.com/tomerovadia/VoterHelpLine/blob/master/src/slack_interaction_handler.ts#L271

Reply:
switched to the helper.

no need to replace blocks since this is when we're creating them for the first time